code/modules/projectiles/projectile/hookshot.dm

```diff
@@ -144,3 +148,13 @@
 		else
 			hookshot.rewind_chain()					//hit something that we can neither pull ourselves to nor drag to us? Just retract the chain.
 	bullet_die()
+
+
+/obj/item/projectile/gravitywell/cultify()
```

Comment:
Why not code/modules/projectiles/projectile/gravitywell.dm?

Also do the same thing with shuttle_act() to make gravitywells undestroyable by shuttles